fwb-plugin/src/test/java/sub/fwb/URLTest.java

```diff
@@ -0,0 +1,61 @@
+package sub.fwb;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.net.URLConnection;
+
+import org.apache.commons.codec.binary.Base64;
+
+public class URLTest {
```

Comment:
This is not a real Test, but more a standalone application?
Better integrate with the existing test frameworks.